internal/dag/gatewayapi_processor.go

```diff
@@ -194,44 +197,44 @@ func (p *GatewayAPIProcessor) computeHTTPRoute(route *gatewayapi_v1alpha1.HTTPRo
 			case gatewayapi_v1alpha1.PathMatchPrefix:
 				pathPrefixes = append(pathPrefixes, stringOrDefault(match.Path.Value, "/"))
 			default:
-				p.Error("NOT IMPLEMENTED: Only PathMatchPrefix is currently implemented.")
+				routeAccessor.ConditionFor(status.Invalid, "PathMatchType", "HTTPRoute.Spec.Rules.PathMatch: Only Prefix match type is supported.")
```

Comment:
Do we want to pull the reasons into constants as well or since these are temporary we're ok with leaving them as is?

Reply:
Same logic as the conditions I guess, constraining what could be passed in with a type

Reply:
In general, I agree that we should make a set of defined Reasons for any condition we're using. As you say, because this is temporary though, we can probably not worry for now. Depends on how we update `ConditionFor`.